cpu/stm32_common/periph/dac.c

```diff
@@ -9,7 +9,7 @@
 
 /**
  * @ingroup     cpu_cortexm_common
```

Comment:
not sure. Seems like someone changed it to this group for all shared stm periph drivers, so I figured it should be like this?!